ui/src/app/tasks-jobs/tasks/create/create.component.html

```diff
@@ -25,31 +29,33 @@ <h3 class="modal-title">Create task</h3>
       <div *ngIf="errors && errors.length > 0" class="alert alert-error">
         <div *ngFor="let error of errors">• {{ error }}</div>
       </div>
-      <div style="margin-bottom: 0.8rem">This action will create the following <strong>task</strong>:</div>
+      <div style="margin-bottom: 0.8rem">
+        <div [innerHTML]="'tasks.create.modal.content' | translate"></div>
+      </div>
       <div class="alert alert-danger" *ngIf="form.hasError('uniqueStreamNames')">
         <div class="alert-items">
           <div class="alert-item static">
             <div class="alert-icon-wrapper">
               <clr-icon class="alert-icon" shape="exclamation-circle"></clr-icon>
             </div>
-            <div class="alert-text">Duplicate task name, please check input names.</div>
+            <div class="alert-text">{{ 'tasks.create.modal.duplicateName' | translate }}</div>
           </div>
         </div>
       </div>
       <div class="clr-form-control clr-row" style="margin-top: 0; max-height: 16px">
-        <label class="clr-control-label clr-col-12 clr-col-md-2">Definition</label>
+        <label class="clr-control-label clr-col-12 clr-col-md-2">{{ 'tasks.main.definition' | translate }}</label>
```

Comment:
Texts like `Definition`, `Name` and `Task Name` are visible on other pages.
Can we move these to `commons` translations ?